src/main/java/com/authlete/jaxrs/BaseResourceEndpoint.java

```diff
@@ -129,7 +129,45 @@ public AccessTokenInfo validateAccessToken(AuthleteApi api, String accessToken)
     public AccessTokenInfo validateAccessToken(
             AuthleteApi api, String accessToken, String[] requiredScopes) throws WebApplicationException
     {
-        return validateAccessToken(api, accessToken, requiredScopes, null);
+        return validateAccessToken(api, accessToken, requiredScopes, null, null);
+    }
+
+
+    /**
+     * Validate an access token. This method is an alias of {@link
+     * #validateAccessToken(AuthleteApi, String, String[], String)
```

Comment:
`#validateAccessToken(AuthleteApi, String, String[], String)` to
`#validateAccessToken(AuthleteApi, String, String[], String, String)`